packages/amplify_core/android/build.gradle

```diff
@@ -43,6 +43,6 @@ android {
 }
 
 dependencies {
-    implementation 'com.amplifyframework:core:1.4.1'
+    implementation 'com.amplifyframework:core:1.6.0'
```

Comment:
Can we update our android dependencies to 1.6.0 without updating all of our other category plugins to use 1.6.0 as well?  I imagine just updating one module could cause issues when multiple modules with different dependency versions are used together .

Reply:
The latest version is 1.6.2, so please update to that.  1.6.1 specifically has a lot of changes specific for Flutter support.

Reply:
Yes that will be part of the rebase.